src/modules/ejector/ejector.py

```diff
@@ -130,6 +137,12 @@ def get_validators_to_eject(self, blockstamp: ReferenceBlockStamp) -> list[tuple
 
         return validators_to_eject
 
+    @lru_cache(maxsize=1)
+    def _is_paused(self, blockstamp: ReferenceBlockStamp) -> bool:
```

Comment:
Should it be called with latest blockstamp?
Should we stop sending transactions in this case?

Reply:
`build_report` is run over the given block stamp and cached, so I preserve this behavior.

Reply:
We should not exit validators if on the reference slot the oracle contract was on pause. We may skip the check on the latest block since a tx will be reverted, but if we can check it and log the result it's better to do it

Reply:
Yeah, maybe let's check it via latest_blockstamp, or just do not provide block identifier to call
and remove cache.
Otherwise, we will have in logs a lot of reverted transaction and grafana will be a little bit broken

Reply:
Changed to a call to `isPaused` with no block identifier at the top of `execute_module` method.